editor/plugins/script_editor_plugin.h

```diff
@@ -393,7 +394,10 @@ class ScriptEditor : public PanelContainer {
 
 	bool open_textfile_after_create = true;
 	bool trim_trailing_whitespace_on_save;
+	bool format_on_save;
+	bool use_space_indentation;
 	bool convert_indent_on_save;
+	void _format_code(ScriptEditorBase *se);
```

Comment:
```suggestion
	void _format_code(ScriptEditorBase *p_se);
```
Prefix arguments with `p_`